### DISCUSSION TITLE
Welcome to mern_youtube_clone Discussions!

### DISCUSSION THREAD

**Discussion by mnnkhndlwl** [mnnkhndlwl/mern_youtube_clone#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Regis-Loyaute** [comment]
Hello, i'm planning to adapt your project in order so store everything localy, how would you approach that ?
@mnnkhndlwl

**Comment by mnnkhndlwl** [comment]
https://youtu.be/wIOpe8S2Mk8 to store files locally you can refer to this video

**Reply by Regis-Loyaute** [reply]
Thanks i'll take a look
